### DISCUSSION TITLE
How can i add direction lines on my map ?

### DISCUSSION THREAD

**Discussion by azimari-toure-ikbal** [uber/react-map-gl#2290] [Q&A]
Hello everyone ! This is my map and I want to be able to create lines from point A to point B passing by the markers. So how can I draw lines between two markers please ? This will look like Google Directions API

<img width="1073" alt="image" src="https://github.com/visgl/react-map-gl/assets/119160287/6abb07e8-6b16-49f1-84b6-bcee133bff5f">

**Comment by gregegan** [comment]
I also am looking to build something similar and looking for recommended path.

I also came across this old thread on a similar topic
https://github.com/visgl/react-map-gl/issues/591

**Reply by gregegan** [reply]
Here is another thread https://github.com/visgl/react-map-gl/discussions/1379#discussioncomment-412539

**Reply by azimari-toure-ikbal** [reply]
Thank you ! I'll have a look but this is definitely complicated.
